1.generate-profiles/profiling_config.yaml

```diff
@@ -64,3 +65,14 @@ aggregate:
     guide:
       - Metadata_Foci_Barcode_MatchedTo_GeneCode
       - Metadata_Foci_Barcode_MatchedTo_Barcode
+---
+normalize:
+  perform: true
+  output_basedir: data/profiles
```

Comment:
Everywhere else so far we've set config defaults to single folders instead of nested folders.
I don't necessarily think this is a problem, just wanted to point it out.

Reply:
great catch - yes this is definitely something to be mindful of. Fortunately, [`pathlib` is great](https://treyhunner.com/2019/01/no-really-pathlib-is-great/) and will take care of that issue under the hood.

Reply:
Also note that we are modeling the directory structure in this project from https://github.com/cytomining/profiling-handbook/issues/54#issue-610880499